src/figuresClasses.ts

```diff
@@ -1,19 +1,102 @@
+// import { moveEmitHelpers } from 'typescript';
+
 export interface Figure {
+  color: 'red' | 'green' | 'blue';
+  shape: 'triangle' | 'circle' | 'rectangle';
 
+  getArea: () => number;
 }
 
 export class Triangle {
+  shape: string = 'triangle';
```

Comment:
you can omit the string type as you declare triangle type

Reply:
have added enum for this line :)